docs/api/ratsd.yaml

```diff
@@ -0,0 +1,148 @@
+openapi: 3.0.0
+info:
+  title: RATS Evidence Collection Daemon
+  version: 0.0.1
+tags: []
+paths:
+  /ratsd/chares:
```

Comment:
I suggest we rename this to a more clear short version as chalrsp
which is more intuitive of what you intend to do.. Personally I struggled with what it means to be a chares?

Reply:
Why not challengeReponse in full?  ( camelCase or whatever notwithstanding?)   

Is this interface only to be used for authentication tokens?

Reply:
> Is this interface only to be used for authentication tokens?

What do you mean by "authentication tokens"?

Reply:
> Personally I struggled with what it means to be a chares?


[Chares](https://en.wikipedia.org/wiki/Chares_of_Lindos) of Lindos, the sculptor of one of the seven wonders of the world!

Reply:
> > Is this interface only to be used for authentication tokens?
> 
> What do you mean by "authentication tokens"?

Just referring to an earlier statement.  What is expected in the POST body and what is returned?

Reply:
> What is expected in the POST body and what is returned?

The request body has an `application/vnd.veraison.chares+json` typed object:

```json
{ "nonce": "base64url_encoded_nonce_" }
```

The request will also sport the auth token in the `Authorization` header:

```http
Authorization: Bearer my.jwt.token
```

The response is a signed EAT with profile `tag:github.com,2024:veraison/ratsd`